tools/lint-all

```diff
@@ -19,6 +19,7 @@ tools = {
     "Type consistency (mypy)": "./tools/run-mypy",
     "PEP8 & more (flake8)": ["flake8"] + python_sources,
     "Formatting (black)": ["black", "--check"] + python_sources,
+    "Docstring generation (developer-file-overview is generated using docstrings)": "./tools/lint-docstring",
```

Comment:
I wouldn't expect something which modifies things to be a linter, so save this for the next commit only.